python/paddle/fluid/layers/nn.py

```diff
@@ -326,6 +327,8 @@ def embedding(input,
     """
 
     helper = LayerHelper('embedding', **locals())
+    if remote_prefetch:
+        assert is_sparse is True and is_distributed is False
```

Comment:
So strange...is_distributed must be false to remote prefetch...Is remote_prefetch distributed or not?

Reply:
remote_prefetch  is deleted from the interface.